packages/story-editor/src/components/library/panes/pageTemplates/pageTemplate.js

```diff
@@ -120,6 +150,48 @@ function PageTemplate(
     setIsHover(false);
   }, []);
 
+  const imageUrl = page.image?.url ? page.image.url : pageDataUrl;
```

Comment:
nit: this would work too
```suggestion
  const imageUrl = page.image?.url || pageDataUrl;
```